transfer.py

```diff
@@ -0,0 +1,49 @@
+from pymongo import MongoClient
```

Comment:
I'd like to keep scripts like this that are specific to internal/private data out of the repo. Devs seeing this who do not have/know of Wapo data will be very confused. I'll create a private repo for internal scripts such as this one.